docs/writing-docs/autodocs.md

```diff
@@ -183,6 +183,29 @@ Creating automated documentation with Storybook's Autodocs provides you with the
 
 ## Advanced configuration
 
+### Documenting multiple components
+
+Sometimes it's helpful to document multiple components together. For example, a component library’s ButtonGroup and Button components might not make sense without one another.
+
+Autodocs allows you to document your "main" component, defined by the `component` property, as well as one or more `subcomponents` related to it.
+
+<!-- prettier-ignore-start -->
+
+<CodeSnippets
+  paths={[
+    'common/subcomponents-in-meta.js.mdx',
+    'common/subcomponents-in-meta.ts.mdx'
+  ]}
+/>
+
+<!-- prettier-ignore-end -->
+
+![Subcomponents in ArgTypes doc block](./doc-block-arg-types-subcomponents.png)
```

Comment:
Can you confirm that this image is accurate? I suspect this was documented there as a placeholder and should be updated, as it was captured from a previous example and should instead align with the above example.

Reply:
Good eye! Looks like it's always been wrong. 😅 

https://storybook.js.org/docs/6.5/writing-docs/docs-page#subcomponents-parameter

Reply:
I resolved this by simply re-using the List-based snippet and screenshot.

Reply:
Let's go with it then.